projects/dvl-fw/src/lib/plugins/default/csv-template-project.ts

```diff
@@ -88,7 +96,7 @@ export class CSVTemplateProject extends DefaultProject {
         labelPlural: 'CSV Data',
         description: fileName || undefined,
         defaultRecordStream: 'csvData',
-        dataVariables: this.fields.map(f => {
+        dataVariables: this.getFilteredFields(this.fields).map(f => {
```

Comment:
This doesn't add data variables for columns with "xxx$$yyy" where xxx is not in the list of fields. Ie if they mispelled xxx or they just want to have certain gv's and nothing else for a dv. In this case a new field xxx should be added. Setting up GVs should still stay the same.